TYPO3.Neos/Classes/TYPO3/Neos/Service/Mapping/NodePropertyConverterService.php

```diff
@@ -0,0 +1,248 @@
+<?php
+namespace TYPO3\Neos\Service\Mapping;
+
+/*
+ * This file is part of the TYPO3.Neos package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use TYPO3\Flow\Annotations as Flow;
+use TYPO3\Flow\Object\ObjectManagerInterface;
+use TYPO3\Flow\Property\PropertyMapper;
+use TYPO3\Flow\Property\PropertyMappingConfiguration;
+use TYPO3\Flow\Property\PropertyMappingConfigurationInterface;
+use TYPO3\Flow\Reflection\ObjectAccess;
+use TYPO3\Flow\Utility\TypeHandling;
+use TYPO3\TYPO3CR\Domain\Model\NodeInterface;
+use TYPO3\TYPO3CR\Domain\Model\NodeType;
+
+/**
+ * Creates PropertyMappingConfigurations to map NodeType properties for the Neos interface.
+ *
+ * @Flow\Scope("singleton")
+ */
+class NodePropertyConverterService
+{
+    /**
+     * @Flow\InjectConfiguration(package="TYPO3.Neos", path="userInterface.inspector.dataTypes")
+     * @var array
+     */
+    protected $typesConfiguration;
+
+    /**
+     * @Flow\Inject
+     * @var ObjectManagerInterface
+     */
+    protected $objectManager;
+
+    /**
+     * @Flow\Inject
+     * @var PropertyMapper
+     */
+    protected $propertyMapper;
+
+    /**
+     * @Flow\Transient
+     * @var array
+     */
+    protected $generatedPropertyMappingConfigurations = array();
+
+    /**
+     * Get a single property reduced to a simple type representation
+     *
+     * @param NodeInterface $node
+     * @param $propertyName
+     * @return string
+     */
+    public function getProperty(NodeInterface $node, $propertyName)
+    {
+        return $this->getPropertyInternal($node, $propertyName);
+    }
```

Comment:
hmm why two methods when this one just calls the internal one? wouldn't this one suffice?